app/js/controllers/room_controller.coffee

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+angular.module 'tandemApp'
+
+  .controller 'RoomController', ($scope, Room) ->
+    $scope.title = "Rooms"
+
+    $scope.rooms = Room.getRooms()
+    $scope.addRoom = ->
+      if $scope.newRoom
+        newRoom =
+          name: $scope.newRoom.name
+          calendarUrl: $scope.newRoom.calendarUrl
+
+        Room.addRoom(newRoom).$promise.then ->
+          # TODO: add success message
+          $scope.rooms.push newRoom
+          $scope.newRoom = {}
+
+        .catch ->
+          $scope.rooms.push newRoom
+          # TODO: Error message
+
+    $scope.deleteRoom = (index) ->
+      x = window.confirm('Are you sure you want to delete this room?')
```

Comment:
Can we change this variable from x to something meaningful? I know it is like that on our mentor one, but its not very descriptive.